### ISSUE TITLE
Link Example

### ISSUE THREAD

**Issue by jlborin** [dequelabs/Deque-University-for-Android#41]
"Click Here" links don't let the user determine the purpose of the link based on the link text alone. If there are multiple "Click Here" links inside a text view, then an Android TalkBack user cannot determine which link goes to which destination.
